index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="header">
+            <a href="/" >
+                <img src="images/ic_pandamarket_logo.png" alt="판다마켓 로고" class="logo"></a>
```

Comment:
💊 제안

현재 코드에서는 <a> 태그와 <img> 태그가 들여쓰기 없이 한 줄로 붙어 있어 가독성이 조금 떨어질 수 있습니다.
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
            <a href="/" >
                <img src="images/ic_pandamarket_logo.png" alt="판다마켓 로고" class="logo">
            </a>
```